funnel/assets/js/app.js

```diff
@@ -14,6 +14,7 @@ $(() => {
   Utils.collapse();
   Utils.smoothScroll();
   Utils.navSearchForm();
+  Utils.scrollTabs();
```

Comment:
Tab scroll function call was included back that was removed accidentally in a previous PR.